flink-connectors/flink-connector-pulsar/src/main/java/org/apache/flink/connector/pulsar/sink/writer/PulsarWriter.java

```diff
@@ -124,8 +131,17 @@ public PulsarWriter(
             throw new FlinkRuntimeException("Cannot initialize schema.", e);
         }
 
+        // Initialize metrics
+        this.pulsarSinkWriterMetricUtils =
```

Comment:
The naming is a bit confusing, it should not be utils tho, need to come up with a better naming